app/views/browse/index.html.erb

```diff
@@ -19,14 +19,12 @@
 <%= render "shared/browse_header", { margin_bottom: 9 } do %>
   <%= render "govuk_publishing_components/components/heading", {
     text: t("browse.title"),
-    inverse: true,
     font_size: "xl",
```

Comment:
Looks good to me, we would also want to apply the same change to the [second level browse page template](https://github.com/alphagov/collections/blob/main/app/views/second_level_browse_page/show_curated.html.erb#L20-L33), for example - https://collections-pr-3985.herokuapp.com/browse/driving/clean-air-zones-emissions-electric-vehicles

Reply:
Oh great, thanks! I've updated the `show_a_to_z.html` and `show_curated.html` views and merged into the 'Update browse header style' commit.